src/helpers/Host.js

```diff
@@ -0,0 +1,53 @@
+import Peer from 'peerjs';
+import EventBus from './EventBus';
+
+let instance = null;
+
+/**
+ * P2P Schema
+ * {
+ *    event: <string>,
+ *    source: <string>,
+ *    metaData: <nested-json-specific-for-command>
+ * }
+ */
+
+export default class Host {
+  constructor() {
+    this.clientConnections = [];
+    this.EventBus = EventBus.getInstance();
+    this.peer = new Peer();
+
+    this.peer.on('open', id => {
+      this.id = id;
+      console.log(id);
+      this.peer.on('connection', conn => {
+        this.clientConnections.push(conn);
+        // Setting up listener for incoming
+        // TODO: Maybe move this to a webworker
+        conn.on('data', data => this.handlePeerData(data));
```

Comment:
Are there any other events we should listen to, like for errors or disconnects?

Reply:
probably. I figured I would add reconnect logic and disconnect logic later